README.md

```diff
@@ -9,10 +19,20 @@
 - Install requirements `pip3 install -r requirements.txt`
 - `pre-commit install`
 
-## Update versions
+### Update Versions
 
 `pip-compile --output-file=requirements.dev.txt requirements.dev.in --upgrade`
 
-# Run `pre-commit` locally.
+### Run `pre-commit` Locally.
 
 `pre-commit run --all-files`
+
+## Setup for Users
+
+- Clone repo: `git clone -b Assignment_1 https://github.com/wpmcgrath95/BDA696_ML_Project.git`
+  - **Note**: Make sure you are in an empty directory, otherwise type `mkdir newdir`
+- Go into the directory where the repo is cloned: `cd newdir`
+- Enter into the BDA696_ML_Project directory: `cd BDA696_ML_Project`
+- Run script: `./scripts/run-ml-code.sh`
+  - **Note**: If you are unable to run script, try typing `chmod +x ./scripts/run-ml-code.sh` and then  
+     `./scripts/run-ml-code.sh`
```

Comment:
This worked perfectly! I think I will try this when I fix my code.

Reply:
yay! That's great.